src/Microsoft.OData.Client/IHttpClientHandlerProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="IHttpClientHandlerProvider.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System.Net.Http;
+
+namespace Microsoft.OData.Client
+{
+    /// <summary>
+    /// Provides the <see cref="HttpClientHandler"/> instance(s) used by OData Client
+    /// to execute requests.

```

Comment:
is the `HttpcClienthHandler` used to execute requests? can the documentation be improved to mirror what the `HttpClienthHandler` does?

Reply:
Technically, the `HttpClientHandler` does execute the requests. The `HttpClient`' `SendAsync` method, calls the `HttpClientHandler`'s `SendAsync` method.